nikola/plugins/command/new_post.py

```diff
@@ -243,6 +243,7 @@ def _execute(self, options, args):
         date_path_opt = options['date-path']
         date_path_auto = self.site.config['NEW_POST_DATE_PATH'] and content_type == 'post'
         date_path_format = self.site.config['NEW_POST_DATE_PATH_FORMAT'].strip('/')
+        post_type = options['type'] if str('type') in options else 'text'
```

Comment:
Why not `if 'type' in options`?

Reply:
Ah! My IDE was reporting that `type` clashes with a built-in. The other way works too, but I tried it this way to see if it removed the warning, which it did. I can change it back if you want.

Reply:
You might have mixed up a `type` variable and a `type` string, please change to a regular string in the `if` condition.

Reply:
I think that's what it was. I've made the change.